journald-logger.gemspec

```
@@ -12,19 +10,11 @@ Gem::Specification.new do |spec|
   spec.license = "MIT"
 
   spec.files = `git ls-files -z`.split("\x0")
-  spec.test_files = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ["lib"]
 
-  spec.required_ruby_version = ">= 2.5.0"
+  spec.required_ruby_version = ">= 3.2.0"
```

Here 3.1 please. RHEL9 has 3.1 and 3.3 Ruby and this is important for us.

---

By default RHEL 9 has Ruby 3.0 which is also what Foreman is using now
```suggestion
  spec.required_ruby_version = ">= 3.0"
```